### DISCUSSION TITLE
What damages can AI models cause to hardware/software?

### DISCUSSION THREAD

**Discussion by andrea3x** [Jeffser/Alpaca#277] [General]
Hello!
Citing the README:
> This project is not affiliated at all with Ollama, I'm not responsible for any damages to your device or software caused by running code given by any AI models.

I'm not that much into AI models, so can you please elaborate on the damages that AI models can do to my hardware/software?

Thanks in advance!

**Comment by Jeffser** [comment]
Hi, models by themselves can't damage your software or hardware, but they can generate code that could do that if you run it without knowing what it does.

For example, let's say a model hallucinates and gives you this line of code

⚠️DO NOT RUN UNDER ANY CIRCUMSTANCES⚠️
```BASH
#sudo rm -rf /
```

That command deletes everything in your system, some distros might stop it but others might not.

That's what I'm trying to convey with these warnings, is not that models, Ollama or Alpaca are dangerous, is that they could output some malicious code and I don't want people blaming me or Ollama for any data or hardware lost by running stuff without knowing what it does.

Hope this clarifies the warning!

**Reply by andrea3x** [reply]
Thank you for the clarification!
It all makes sense now.
Have a nice day!
